models/gas/ethereum/gas_ethereum_fees_traces.sql

```diff
@@ -0,0 +1,57 @@
+{{ config(
+    alias = 'fees_traces',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash', 'trace'],
+    )
+}}
+
+SELECT 'ethereum' AS blockchain
+, etg.block_time
+, date_trunc('day', etg.block_time) AS block_date
+, etg.block_number
+, etg.tx_hash
+, MAX(etg.to) AS trace_to
+, etg.trace
+, SUM(etg.gas_used) AS gas_used
+, (SUM(etg.gas_used)*etc.gas_price)/1e18 AS spent_gas_fee
+, (AVG(pu.price)*SUM(etg.gas_used)*etc.gas_price)/POWER(10, 18) AS spent_gas_fee_usd
+FROM (
+     SELECT et.to 
+     , et.tx_hash
+     , et.trace_address AS trace
+     , et.gas_used
+     , et.block_time
+     , et.block_number
+     FROM {{ source('ethereum','traces') }} et
```

Comment:
Can we also add `trace_method`
bytearray_substring(input,1,4)

& if it's not overkill, join to transactions table to pull:
- tx_from
- tx_to
- tx_method `bytearray_substring(data,1,4)`